SecurityAgent/AccessControlList.h

```diff
@@ -40,8 +40,7 @@ namespace {
         string regex = input;
         
         // order of replacing is important
-        ReplaceString(regex,"*","^[a-zA-Z0-9.]+$");
-        ReplaceString(regex,".","\\.");
```

Comment:
dot should be treated as dot

Reply:
Thanks for the review @npoltorapavlo .

`CreateRegex() ` is used for generating regex for method/plugin names only. So that's why I was expecting `.` to be treated as a wildcard character.
So if `".nfo"` is added to the methods to allow, was expecting `"Info"` to be allowed (and was not expecting a method with name `".nfo"` )

But for URL parsing case, agree that `.` should be treated as literal dot itself. `CreateUrlRegex()` is used for creating regex for URLs and it has the logic to treat `.` as literal.
`ReplaceString(regex,".","\\.");`

This was my idea when making these changes. Please correct me if I am wrong.

Reply:
in acl `*` is wildcard. i.e. you can use "*nfo". in acl we have callsigns like `"org.rdk.Network"` (with dots)

Reply:
Agree.. Thanks for the clarification @npoltorapavlo